lib/paperclip/globalize3/attachment.rb

```diff
@@ -43,18 +43,32 @@ def clear_with_globalize3(*args)
         end
 
         private
+        # If translated and locale specific options are present, check if those locales really exists and remove
+        # missing ones. If none of the locales are existing do nothing, otherwise execute to block with existing
+        # locales only
+        def without_missing_translations(options, &block)
```

Comment:
Could we put this functionality into `with_locales_if_translated`? It's always used together and I think it would make sense to combine it.